CMakeModules/SplitDebugInfo.cmake

```diff
@@ -0,0 +1,88 @@
+# Tailored after https://github.com/GerbilSoft/mcrecover/blob/master/cmake/macros/SplitDebugInformation.cmake
+# Minor modifications to original
+
+if (NOT MSVC)
+  include(CMakeFindBinUtils)
+  if (NOT APPLE AND NOT CMAKE_OBJCOPY)
+    message(WARNING "'objcopy' was not found; debug information will not be split.")
+  elseif (NOT CMAKE_STRIP)
+    message(WARNING "'strip' was not found; debug information will not be split.")
+  endif ()
+endif (NOT MSVC)
+
+function(af_split_debug_info _target)
+  set(SPLIT_TOOL_EXISTS ON)
+  if (MSVC)
+    set(SPLIT_TOOL_EXISTS OFF)
+    if(AF_PACK_DEBUG_SYMBOLS AND WIN32)
+      install(FILES
+        $<TARGET_PDB_FILE:${_target}>
+        DESTINATION ${AF_INSTALL_LIB_DIR}
+        COMPONENT "${_target}_debug_symbols"
+        )
+    endif()
+  elseif (NOT APPLE AND NOT CMAKE_OBJCOPY)
+    set(SPLIT_TOOL_EXISTS OFF)
```

Comment:
These two else statements do the same thing. is that intended?

Reply:
Yes, we are looking for OBJCOPY only on Linux style distributions.